tools/ci_connector_ops/ci_connector_ops/pipelines/contexts.py

```diff
@@ -175,19 +179,7 @@ def get_repo_dir(self, subdir: str = ".", exclude: Optional[List[str]] = None, i
         return self.dagger_client.host().directory(subdir, exclude=exclude, include=include)
 
     def create_slack_message(self) -> str:
-        message = f"*<{self.gha_workflow_run_url}|{self.pipeline_name}>*\n"
-        message += f"*Branch:* {self.git_branch}\n"
-        message += f"*Commit:* {self.git_revision[:10]}\n"
-
-        if self.state in [ContextState.INITIALIZED, ContextState.RUNNING]:
-            message += "🟠"
-        if self.state is ContextState.SUCCESSFUL:
-            message += "🟢"
-        if self.state is ContextState.FAILURE:
-            message += "🔴"
-        message += f" {self.state.value['description']}"
-
-        return message
+        raise NotImplementedError()
```

Comment:
I prefer to keep implementation of slack messages for other contexts (PipelineContext, ConnectorContext) for later.